.github/workflows/db-release.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow updates the database and creates a release of it everyday
+
+name: Release Workflow
+on:
+  schedule:
+  - cron: "0 0 * * *"
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'src/tech_list.csv'
+
+jobs:
+  db-release:
+    name: "Database Release"
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 30
```

Comment:
Why do you hardcode the `timeout-minutes` variable here? It seems that the default value is 360 (https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) and is way better for our use-case